backend/packages/Upgrade/src/api/repositories/ExperimentAuditLogRepository.ts

```diff
@@ -1,28 +1,29 @@
 import { ExperimentAuditLog } from '../models/ExperimentAuditLog';
-import { EntityRepository, Repository } from 'typeorm';
+import { EntityRepository, Repository, EntityManager } from 'typeorm';
 import { EXPERIMENT_LOG_TYPE } from 'upgrade_types';
 import { User } from '../models/User';
 import repositoryError from './utils/repositoryError';
 
 @EntityRepository(ExperimentAuditLog)
 export class ExperimentAuditLogRepository extends Repository<ExperimentAuditLog> {
-  public async paginatedFind(limit: number, offset: number, filter: EXPERIMENT_LOG_TYPE): Promise<ExperimentAuditLog[]> {
+  public async paginatedFind(
+    limit: number,
+    offset: number,
+    filter: EXPERIMENT_LOG_TYPE
+  ): Promise<ExperimentAuditLog[]> {
     let queryBuilder = this.createQueryBuilder('audit')
       .skip(offset)
       .take(limit)
       .leftJoinAndSelect('audit.user', 'user')
       .orderBy('audit.createdAt', 'DESC');
 
     if (filter) {
-      queryBuilder = queryBuilder
-        .where('audit.type = :filter', { filter });
+      queryBuilder = queryBuilder.where('audit.type = :filter', { filter });
     }
-    return queryBuilder
-      .getMany()
-      .catch((error: any) => {
-        const errorMsg = repositoryError('ExperimentAuditLogRepository', 'paginatedFind', { limit, offset }, error);
-        throw new Error(errorMsg);
-      });
+    return queryBuilder.getMany().catch((error: any) => {
```

Comment:
@RidhamShah No new code changes here but formatting is updated. Can you please revert the changes?